styles.css

```diff
@@ -1,146 +1,1172 @@
-
 @import url('https://fonts.googleapis.com/css2?family=Poppins&family=Roboto&display=swap');
 
-.first-page {
+.second-paper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+  position: absolute;
+  width: 375px;
+  height: 2788px;
+  left: 0;
+  top: 28px;
+}
```

Comment:
- [ ] Please, as required in this project milestone, use both `display: grid` and `display: flex` to align the items. For all the projects, the container uses a grid, and for individual projects uses flex. Kindly work on this :+1:

Reply:
Good job following the advice of the previous reviewer